packages/css/src/components/footer/footer.scss

```diff
@@ -3,6 +3,55 @@
  * Copyright Gemeente Amsterdam
  */
 
+@use "../../common/text-rendering" as *;
+
+@mixin reset-ul {
+  margin-block: 0;
+  padding-inline: 0;
+}
+
 .ams-footer__top {
```

Comment:
We decided that `.ams-footer` was going to set horizontal padding just like the new Header does, and that people don’t need to include a `Grid` in the footer anymore.